reference/configuration/doctrine.rst

```diff
@@ -31,9 +31,15 @@ Full Default Configuration
                         port:                 ~
                         user:                 root
                         password:             ~
+                        # charset of the database
                         charset:              ~
+                        # charset and collation of the tables. Not inherited from database
+                        default_table_options:
+                            charset:          ~
+                            collate:          ~
                         path:                 ~
                         memory:               ~
+                        
```

Comment:
just a minor comment: this blank line should be removed (but we can do that while merging)